cmd/confighandler.go

```diff
@@ -222,7 +222,7 @@ func (conf *DeviceConfigHandler) CreateDeviceFromSpec(deviceDef string) {
 
 	// If this is an RTU over TCP device, a default RTU over TCP should already
 	// have been created of the --rtu flag was specified. We'll not re-check this here.
-	manager := conf.ConnectionManager(connSpec, false, 0, "")
+	manager := conf.ConnectionManager(connSpec, false, 0, "", 300)
```

Comment:
Did you test this? 300 is an int. Would suggest to make

    DefaultTimeout = 300 * time.Millisecond

a global const.